src/graphics/gfx_update.cpp

```diff
@@ -2275,6 +2271,9 @@ void UpdateGraphics(bool skipRepaint)
                 XRender::offsetViewportIgnore(true);
                 SuperPrint(fmt::format_ne("{0}", int(PrintFPS)), 1, 8, 8, 0.f, 1.f, 0.f);
 
+#if 0
+                // deprecated profiling info
+
```

Comment:
You better put the comment at the `#if 0` itself, like:
```cpp
#if 0 // deprecated profiling info
```

Reply:
Good note, done.